### ISSUE TITLE
Suggestion: a stack configuration for easier installation

### ISSUE THREAD

**Issue by Athas** [AccelerateHS/accelerate#391]
The Accelerate website has a [Get Started](http://www.acceleratehs.org/get-started/linux) guide that uses a Cabal-based workflow.  Following the instructions didn't work for me for usual Cabal-related mystery reasons (a type error in the `fclabels-2.0.3.2` package), probably because the versions involved are not entirely fixed.  The following `stack.yaml` was a good starting point for me:

```
flags: {}
extra-package-dbs: []
packages: []
extra-deps:
        - accelerate-llvm-1.0.0.0
        - llvm-hs-4.0.1.0
        - llvm-hs-pure-4.0.0.0
        - cuda-0.7.5.3
resolver: lts-9.0
```

I still had to manually use `--flag=llvm-hs:shared-llvm` on the `stack` command line, though.  I don't have enough Stack-fu to know whether this can be embedded into the configuration.  It probably can.

**Comment by Athas** [comment]
This one has been sufficient for installing `accelerate-examples`:

```
flags: {}
extra-package-dbs: []
packages: []
extra-deps:
        - accelerate-llvm-1.0.0.0
        - llvm-hs-4.0.1.0
        - llvm-hs-pure-4.0.0.0
        - cuda-0.7.5.3
        - accelerate-fft-1.0.0.0
        - accelerate-io-1.0.0.0
        - accelerate-llvm-native-1.0.0.0
        - accelerate-llvm-ptx-1.0.0.1
        - colour-accelerate-0.1.0.0
        - gloss-accelerate-2.0.0.0
        - gloss-raster-accelerate-2.0.0.0
        - lens-accelerate-0.1.0.0
        - normaldistribution-1.1.0.3
        - repa-3.4.1.2
        - repa-io-3.4.1.1
        - vector-0.11.0.0
        - cufft-0.7.5.0
        - primitive-0.6.1.0
resolver: lts-9.0
```

**Comment by tmcdonell** [comment]
Hey @Athas, sorry I didn't see this.

I don't know what that fclabels problem you had is; 2.0.3.2 is the version included in lts-9.0...  ¯\\_(ツ)_/¯

Anyway, this seems like a good suggestion, I'll add it to the website. By the way, you can add the flags to your `stack.yaml` as such, to avoid manually adding to the command line:

```yaml
flags:
  llvm-hs:
    shared-llvm: true
```
